script/helperFunctions.sh

```diff
@@ -1537,16 +1522,49 @@ function verifyContract() {
       --verifier sourcify
   fi
 
-  if [ $? -ne 0 ]; then
-    # verification apparently failed
-    warning "[info] $CONTRACT on $NETWORK with address $ADDRESS could not be verified using Sourcify"
-    return 1
-  else
-    # verification successful
+  if [ $? -eq 0 ]; then
     echo "[info] $CONTRACT on $NETWORK with address $ADDRESS successfully verified using Sourcify"
     return 0
+  else
+    warning "[info] $CONTRACT on $NETWORK with address $ADDRESS could not be verified using Sourcify"
+    return 1
   fi
 }
+
+function getEtherscanApiKeyName() {
+  local NETWORK="$1"
+  local TOML_FILE="foundry.toml"
```

Comment:
what about having path in config.sh like for networks.json is under NETWORKS_JSON_FILE_PATH then this file would be under FOUNDRY_TOML_FILE_PATH ?

Reply:
Good point :)
While for foundry.toml this is not really necessary (as the file has to be in this very location required by foundry), I still moved it to a variable as it's cleaner coding. You never know what happens.